Makefile

```diff
@@ -3,6 +3,10 @@ DOCKER_IMAGE_BASE=${DOCKER_IMAGE_PREFIX}-base
 DOCKER_IMAGE_QEMUV7=${DOCKER_IMAGE_PREFIX}-qemuv7
 DOCKER_IMAGE_QEMUV8=${DOCKER_IMAGE_PREFIX}-qemuv8
 DOCKER_IMAGE_YOCTO=${DOCKER_IMAGE_PREFIX}-yocto
+DOCKER_IMAGE_BOLOS_EMU=${DOCKER_IMAGE_PREFIX}-bolos-emu
+DOCKER_IMAGE_BOLOS_BUILDER=zondax/ledger-docker-bolos
+DOCKER_IMAGE_CIRCLECI=zondax/circleci
+DOCKER_IMAGE_RUSTCI=zondax/rust-ci
```

Comment:
In the medium term, we should think of refactoring these two images and update any project that depends on them.
I will open an issue for this.